orchagent/orchdaemon.cpp

```diff
@@ -193,7 +193,7 @@ bool OrchDaemon::init()
      * when iterating ConsumerMap.
      * That is ensured implicitly by the order of map key, "LAG_TABLE" is smaller than "VLAN_TABLE" in lexicographic order.
      */
-    m_orchList = { gSwitchOrch, gCrmOrch, gBufferOrch, gPortsOrch, gIntfsOrch, gNeighOrch, gRouteOrch, copp_orch, tunnel_decap_orch, qos_orch, wm_orch, policer_orch };
+    m_orchList = { gSwitchOrch, gCrmOrch, gPortsOrch, gBufferOrch, gIntfsOrch, gNeighOrch, gRouteOrch, copp_orch, tunnel_decap_orch, qos_orch, wm_orch, policer_orch };
```

Comment:
>gPortsOrch [](start = 42, length = 10)

Do not reorder if you don't have a strong reason. Any order should work for warm-reboot if we iterate enough rounds.

Reply:
Agree with Qi

Reply:
@qiluo-msft @prsunny I agree that it is just a matter of enough iterations, however this order makes more sense at least to me, so it's more a cosmetic change.
Besides, now I think with this order 3 instead of 4 iterations are needed in warm boot, let me check.
Do you have concern that this may break something?

Reply:
Let me know if you have proof that 4 iterations -> 3. Otherwise I don't see any improvement.

---
In reply to: [339371158](https://github.com/Azure/sonic-swss/pull/1103#discussion_r339371158) [](ancestors = 339371158)

Reply:
Please check one more commit to have 3 iterations.

Reply:
In any case, let me separate this change in another PR